### ISSUE TITLE
[Bug] Button for virtual controller not showing up

### ISSUE THREAD

**Issue by levram99** [redphx/better-xcloud#582]
### Checklist

- [X] I will only use English in my report.
- [X] I have used the search function for [**open and closed issues**](https://github.com/redphx/better-xcloud/issues?q=is%3Aissue) to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### Questions

- [X] xCloud officially supports my country/region.
- [X] The bug doesn't happen when I disable Better xCloud script.
- [X] The bug didn't happen in previous Better xCloud version (name which one).

### Device

Desktop

### Operating System

Windows 10

### Android app/Browser Version

Brave latest version

### Better xCloud Version

6.0.0

### Game list

Fortnite

### Reproduction Steps

The panel for activate controler emulator is not showing up in recent versions of XCloud version 6.0.0, i have to downgrade to version 4.0.0 to use this function.
 
Using Brave browser latest version.

Better Xcloud version that the panel is working = 4.0.0

### Screenshot/video
![asdsadaddsdasdadadasdas](https://github.com/user-attachments/assets/f967fd3e-bdcd-4c82-84ae-b6cb41223285)

**Comment by redphx** [comment]
Do you have other script/extension installed?

**Comment by levram99** [comment]
No, i only using better xcloud script, and using tampermonkey latest version.
maybe is the browser? i will try it on edge or chrome.

**Comment by redphx** [comment]
check chrome devtools for errors
